test/src/660-store_publish_statistics/main

```diff
@@ -0,0 +1,242 @@
+# This file contains $NR_OF_TESTS tests for checking the store publish statistics feature (checking the database integrity)
```

Comment:
Some questions about the tests:
* Do tests 659 and 660 contain the same checks?
* Are any of the tests run successively on the same repository, to see that rows are properly inserted in an existing db?

Reply:
- Yes
- Yes, in 660 there are 4 tests that run on the same repository.